i18n/en.json

```diff
@@ -255,10 +255,6 @@
     "id": "api.channel.create_channel.direct_channel.app_error",
     "translation": "Must use createDirectChannel API service for direct message channel creation."
   },
-  {
-    "id": "api.channel.create_channel.invalid_character.app_error",
```

Comment:
Please also remove the translations in other files.

Reply:
My assumption was that we only modify `en` translations and all other translations are updated in the community translation PRs as per @jespino 's comment here https://community.mattermost.com/core/pl/jt8sdya5pjfh7bgbr5389gs9ac

Reply:
Hmm .. I am not really aware of this "other process". My main concern was that we are deleting a main translation. So there's not much use of keeping the other translation strings. They are just lying there. If there's another "process" which takes care of deleting those, then I'm fine.

Reply:
Yes, exactly, the translations are added and removed only to the `en.json` file, the rest of the files are updated through our i18n process.